src/widgets/simulation_widget.py

```diff
@@ -13,10 +11,15 @@ def __init__(self):
     def clear_plot(self):
         self.axes.cla()
 
-    def plot_rectangle(self, width, heigth):
-        self.axes.plot([0, width, width, 0, 0], [0, 0, heigth, heigth, 0], color="blue")
-        self.axes.axis("off")
-        self.draw()
+    def plot_region(self, region):
+        x = []
+        y = []
+        # create x and y arrays from vertex array - for plot (can I do oneliner here?)
+        for vertex in region.vertices:
+            x.append(vertex[0])
+            y.append(vertex[1])
```

Comment:
Good question, I found how to do it.
```suggestion
        # create x and y arrays from vertex array - for plot
        x, y = list(zip(*region.vertices))
```